uber/config.py

```diff
@@ -232,12 +232,19 @@ def DEALER_APPS(self):
 
     @request_cached_property
     @dynamic
-    def ATTENDEE_BADGES_SOLD(self):
```

Comment:
This property wasn't actually being used anywhere that I could see.

Reply:
It looks like it was being used in one place, which has now been deleted as part of this MR, so removing this properly seems valid.